Bio/SearchIO/_model/query.py

```diff
@@ -206,72 +205,36 @@ def __init__(self, hits=(), id=None, hit_key_function=None):
+    def __iter__(self):
+        """Iterate over hits."""
+        return iter(self.hits)
 
-        def __iter__(self):
-            """Iterate over hit items."""
-            return self.iterhits()
-
-        @property
-        def hits(self):
-            """Hit objects contained in the QueryResult."""
-            return self._items.values()
-
-        @property
-        def hit_keys(self):
-            """Hit IDs of the Hit objects contained in the QueryResult."""
-            return self._items.keys()
-
-        @property
-        def items(self):
-            """List of tuples of Hit IDs and Hit objects."""
-            return self._items.items()
-
-        def iterhits(self):
-            """Return an iterator over the Hit objects."""
-            yield from self._items.itervalues()  # noqa: B301
-
-        def iterhit_keys(self):
-            """Return an iterator over the ID of the Hit objects."""
-            yield from self._items
-
-        def iteritems(self):
-            """Return an iterator yielding tuples of Hit ID and Hit objects."""
-            yield from self._items.iteritems()  # noqa: B301
-
-    else:
-
-        def __iter__(self):
-            """Iterate over hits."""
-            return iter(self.hits)
-
-        @property
-        def hits(self):
-            """Hit objects contained in the QueryResult."""
-            return list(self._items.values())
+    @property
+    def hits(self):
+        """Hit objects contained in the QueryResult."""
+        return list(self._items.values())
 
-        @property
-        def hit_keys(self):
-            """Hit IDs of the Hit objects contained in the QueryResult."""
-            return list(self._items.keys())
+    @property
+    def hit_keys(self):
+        """Hit IDs of the Hit objects contained in the QueryResult."""
+        return list(self._items.keys())
 
-        @property
-        def items(self):
-            """List of tuples of Hit IDs and Hit objects."""
-            return list(self._items.items())
+    @property
+    def items(self):
+        """List of tuples of Hit IDs and Hit objects."""
+        return list(self._items.items())
 
-        def iterhits(self):
-            """Return an iterator over the Hit objects."""
-            yield from self._items.values()
+    def iterhits(self):
+        """Return an iterator over the Hit objects."""
+        yield from self._items.values()
 
-        def iterhit_keys(self):
-            """Return an iterator over the ID of the Hit objects."""
-            yield from self._items
+    def iterhit_keys(self):
+        """Return an iterator over the ID of the Hit objects."""
+        yield from self._items
 
-        def iteritems(self):
-            """Return an iterator yielding tuples of Hit ID and Hit objects."""
-            yield from self._items.items()
+    def iteritems(self):
+        """Return an iterator yielding tuples of Hit ID and Hit objects."""
+        yield from self._items.items()
```

Comment:
I would be inclined to make this Python 3 dictionary like, remove ``.iteritems()`` and make ``.items()`` an iterator not a list.

Same for the other properties.

Reply:
I don't have a strong opinion about this. I'll leave this to you and @bow for a separate PR.